src/main/java/org/prgrms/cream/domain/product/dto/OptionResponse.java

```diff
@@ -0,0 +1,15 @@
+package org.prgrms.cream.domain.product.dto;
+
+import lombok.Getter;
+
+@Getter
+public class OptionResponse {
+
+	private String size;
+	private int straightBuyPrice;
+
```

Comment:
```suggestion
	private String size;
	private int straightBuyPrice;
	private int straightSellPrice;
```
즉시판매가도 함께 가져오면 좋을 것 같으네요🤔

Reply:
생각치도 못했네유 지적해주셔서 감사합니당👍😆